### PR TITLE
Add task

### PR DESCRIPTION
**Pull request by JuliaZharova**

### CODE REVIEW

File: Lecture-10/src/com/company/Calc.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.company;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Calc<T extends Number> {
```
Comment (ltana) [inline Lecture-10/src/com/company/Calc.java]: use three different types, not one

File: Lecture-10/src/com/company/Calc.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        this.number1 = number1;
+        this.number2 = number2;
+        this.number3 = number3;
+    }
+
+    public double sum () {
+        return number1.doubleValue() + number2.doubleValue() + number3.doubleValue();
+    }
+
+    public <T extends Number> double sum (T number1, T number2, T number3) {
```
Comment (ltana) [inline Lecture-10/src/com/company/Calc.java]: use only one sum method which accepts as arguments three different types, not one

**Review by ltana — CHANGES_REQUESTED** [review]

File: Lecture-10/src/com/company/Calc.java (hunk 1 above)
Comment (JuliaZharova) [reply Lecture-10/src/com/company/Calc.java]: fixed

File: Lecture-10/src/com/company/Calc.java (hunk 2 above)
Comment (JuliaZharova) [reply Lecture-10/src/com/company/Calc.java]: fixed

Comment (ltana) [reply Lecture-10/src/com/company/Calc.java]: types are still the same 'T', please use different ones

**Review by ltana — CHANGES_REQUESTED** [review]

File: Lecture-10/src/com/company/Calc.java (hunk 2 above)
Comment (JuliaZharova) [reply Lecture-10/src/com/company/Calc.java]: fixed

**Review by ltana — APPROVED** [review]
can be merged

**Pull request merged**